servicemesh-install/topics/servicemesh-istio-comparison.adoc

```diff
@@ -45,3 +47,57 @@ spec:
     properties:
       request.regex.headers[<header>]: "<regular expression>"
 ----
+
+== Automatic route creation
```

Comment:
Passive voice in this whole section.

Reply:
I reworked the language in this section.